projects/graph/backup/graph.py

```diff
@@ -157,23 +157,23 @@ def dfs_path(self, starting_node, destination_node):
             s.append(n + [item])
     return False
 
-graph = Graph()
-graph.add_vertex('0')
-graph.add_vertex('1')
-graph.add_vertex('2')
-graph.add_vertex('3')
-graph.add_edge('0', '1')
-graph.add_edge('2', '3')
-graph.add_edge('1', '2')
-print(f'Graph Vertices: {graph.vertices}')
-graph.bft('0')
-graph.dft('0')
-graph.dft_r('0')
-print(graph.bfs('0', '3'))
-print(graph.bfs('0', '4'))
-print(graph.dfs('0', '3'))
-print(graph.dfs('0', '4'))
-print(graph.bfs_path('0', '3'))
-print(graph.dfs_path('0', '3'))
+# graph = Graph()
+# graph.add_vertex('0')
+# graph.add_vertex('1')
+# graph.add_vertex('2')
+# graph.add_vertex('3')
+# graph.add_edge('0', '1')
```

Comment:
what was the purpose of commenting out the test code?

Reply:
There was no provided test code for this project, and no built in way to run the file as I worked on it. I was writing these to see the results of the methods I was creating for my own debugging purposes. I did not want the final file to run these debugging tests in the end though. Just wanted the file to store the class after I knew it was working properly.